[2019-01-18 15:06:15.0565] [WebCore.Startup+<>c__DisplayClass5_0.<Configure>b__9] [Info]: [网站启动] [] [CancellationTokenSource.ExecuteCallbackHandlers => ExecutionContext.RunInternal => <>c__DisplayClass5_0.<Configure>b__9] []
[2019-01-18 15:06:16.0610] [Microsoft.AspNetCore.Server.Kestrel.Https.Internal.HttpsConnectionAdapter+<InnerOnConnectionAsync>d__10.MoveNext] [Debug]: [Failed to authenticate HTTPS connection.] [] [HttpsConnectionAdapter.InnerOnConnectionAsync => AsyncMethodBuilderCore.Start => <InnerOnConnectionAsync>d__10.MoveNext] []
[2019-01-18 15:06:16.0610] [Microsoft.AspNetCore.Server.Kestrel.Https.Internal.HttpsConnectionAdapter+<InnerOnConnectionAsync>d__10.MoveNext] [Debug]: [Failed to authenticate HTTPS connection.] [] [HttpsConnectionAdapter.InnerOnConnectionAsync => AsyncMethodBuilderCore.Start => <InnerOnConnectionAsync>d__10.MoveNext] []
[2019-01-18 15:06:16.9864] [WebCore.Hosting.MyRabbitMqService.<StartAsync>b__5_0] [Info]: [RabbitMq开始监听] [] [Task.ExecuteWithThreadLocal => ExecutionContext.RunInternal => MyRabbitMqService.<StartAsync>b__5_0] []
[2019-01-18 15:08:26.9896] [WebCore.Startup+<>c__DisplayClass5_0.<Configure>b__9] [Info]: [网站启动] [] [CancellationTokenSource.ExecuteCallbackHandlers => ExecutionContext.RunInternal => <>c__DisplayClass5_0.<Configure>b__9] []
[2019-01-18 15:08:28.3685] [WebCore.Hosting.MyRabbitMqService.<StartAsync>b__5_0] [Info]: [RabbitMq开始监听] [] [Task.ExecuteWithThreadLocal => ExecutionContext.RunInternal => MyRabbitMqService.<StartAsync>b__5_0] []
[2019-01-18 15:08:28.3951] [Microsoft.AspNetCore.Server.Kestrel.Https.Internal.HttpsConnectionAdapter+<InnerOnConnectionAsync>d__10.MoveNext] [Debug]: [Failed to authenticate HTTPS connection.] [] [AsyncStateMachineBox`1.MoveNext => ExecutionContext.RunInternal => <InnerOnConnectionAsync>d__10.MoveNext] []
[2019-01-18 15:08:28.3951] [Microsoft.AspNetCore.Server.Kestrel.Https.Internal.HttpsConnectionAdapter+<InnerOnConnectionAsync>d__10.MoveNext] [Debug]: [Failed to authenticate HTTPS connection.] [] [AsyncStateMachineBox`1.MoveNext => ExecutionContext.RunInternal => <InnerOnConnectionAsync>d__10.MoveNext] []
[2019-01-18 15:11:09.7799] [WebCore.Startup+<>c__DisplayClass5_0.<Configure>b__9] [Info]: [网站启动] [] [CancellationTokenSource.ExecuteCallbackHandlers => ExecutionContext.RunInternal => <>c__DisplayClass5_0.<Configure>b__9] []
[2019-01-18 15:11:11.1042] [WebCore.Hosting.MyRabbitMqService.<StartAsync>b__5_0] [Info]: [RabbitMq开始监听] [] [Task.ExecuteWithThreadLocal => ExecutionContext.RunInternal => MyRabbitMqService.<StartAsync>b__5_0] []
[2019-01-18 15:11:11.0964] [Microsoft.AspNetCore.Server.Kestrel.Https.Internal.HttpsConnectionAdapter+<InnerOnConnectionAsync>d__10.MoveNext] [Debug]: [Failed to authenticate HTTPS connection.] [] [AsyncStateMachineBox`1.MoveNext => ExecutionContext.RunInternal => <InnerOnConnectionAsync>d__10.MoveNext] []
[2019-01-18 15:11:11.0747] [Microsoft.AspNetCore.Server.Kestrel.Https.Internal.HttpsConnectionAdapter+<InnerOnConnectionAsync>d__10.MoveNext] [Debug]: [Failed to authenticate HTTPS connection.] [] [AsyncStateMachineBox`1.MoveNext => ExecutionContext.RunInternal => <InnerOnConnectionAsync>d__10.MoveNext] []
[2019-01-18 15:12:24.3828] [Microsoft.AspNetCore.Diagnostics.Internal.DiagnosticsLoggerExtensions.UnhandledException] [Error]: [An unhandled exception has occurred while executing the request.] [] [AsyncMethodBuilderCore.Start => <Invoke>d__7.MoveNext => DiagnosticsLoggerExtensions.UnhandledException] []
[2019-01-18 15:16:01.7205] [WebCore.Startup+<>c__DisplayClass5_0.<Configure>b__9] [Info]: [网站启动] [] [CancellationTokenSource.ExecuteCallbackHandlers => ExecutionContext.RunInternal => <>c__DisplayClass5_0.<Configure>b__9] []
[2019-01-18 15:16:03.0796] [Microsoft.AspNetCore.Server.Kestrel.Https.Internal.HttpsConnectionAdapter+<InnerOnConnectionAsync>d__10.MoveNext] [Debug]: [Failed to authenticate HTTPS connection.] [] [AsyncStateMachineBox`1.MoveNext => ExecutionContext.RunInternal => <InnerOnConnectionAsync>d__10.MoveNext] []
[2019-01-18 15:16:03.0796] [Microsoft.AspNetCore.Server.Kestrel.Https.Internal.HttpsConnectionAdapter+<InnerOnConnectionAsync>d__10.MoveNext] [Debug]: [Failed to authenticate HTTPS connection.] [] [AsyncStateMachineBox`1.MoveNext => ExecutionContext.RunInternal => <InnerOnConnectionAsync>d__10.MoveNext] []
[2019-01-18 15:16:04.1767] [WebCore.Hosting.MyRabbitMqService.<StartAsync>b__5_0] [Info]: [RabbitMq开始监听] [] [Task.ExecuteWithThreadLocal => ExecutionContext.RunInternal => MyRabbitMqService.<StartAsync>b__5_0] []
[2019-01-18 15:19:05.6593] [WebCore.Startup+<>c__DisplayClass5_0.<Configure>b__9] [Info]: [网站启动] [] [CancellationTokenSource.ExecuteCallbackHandlers => ExecutionContext.RunInternal => <>c__DisplayClass5_0.<Configure>b__9] []
[2019-01-18 15:19:06.4595] [Microsoft.AspNetCore.Server.Kestrel.Https.Internal.HttpsConnectionAdapter+<InnerOnConnectionAsync>d__10.MoveNext] [Debug]: [Failed to authenticate HTTPS connection.] [] [HttpsConnectionAdapter.InnerOnConnectionAsync => AsyncMethodBuilderCore.Start => <InnerOnConnectionAsync>d__10.MoveNext] []
[2019-01-18 15:19:08.0369] [WebCore.Hosting.MyRabbitMqService.<StartAsync>b__5_0] [Info]: [RabbitMq开始监听] [] [Task.ExecuteWithThreadLocal => ExecutionContext.RunInternal => MyRabbitMqService.<StartAsync>b__5_0] []
[2019-01-18 15:19:25.6604] [Microsoft.AspNetCore.Diagnostics.Internal.DiagnosticsLoggerExtensions.UnhandledException] [Error]: [An unhandled exception has occurred while executing the request.] [] [AsyncMethodBuilderCore.Start => <Invoke>d__7.MoveNext => DiagnosticsLoggerExtensions.UnhandledException] []
[2019-01-18 15:20:30.0590] [WebCore.Startup+<>c__DisplayClass5_0.<Configure>b__9] [Info]: [网站启动] [] [CancellationTokenSource.ExecuteCallbackHandlers => ExecutionContext.RunInternal => <>c__DisplayClass5_0.<Configure>b__9] []
[2019-01-18 15:20:31.2473] [Microsoft.AspNetCore.Server.Kestrel.Https.Internal.HttpsConnectionAdapter+<InnerOnConnectionAsync>d__10.MoveNext] [Debug]: [Failed to authenticate HTTPS connection.] [] [HttpsConnectionAdapter.InnerOnConnectionAsync => AsyncMethodBuilderCore.Start => <InnerOnConnectionAsync>d__10.MoveNext] []
[2019-01-18 15:20:31.2473] [Microsoft.AspNetCore.Server.Kestrel.Https.Internal.HttpsConnectionAdapter+<InnerOnConnectionAsync>d__10.MoveNext] [Debug]: [Failed to authenticate HTTPS connection.] [] [HttpsConnectionAdapter.InnerOnConnectionAsync => AsyncMethodBuilderCore.Start => <InnerOnConnectionAsync>d__10.MoveNext] []
[2019-01-18 15:20:32.9256] [WebCore.Hosting.MyRabbitMqService.<StartAsync>b__5_0] [Info]: [RabbitMq开始监听] [] [Task.ExecuteWithThreadLocal => ExecutionContext.RunInternal => MyRabbitMqService.<StartAsync>b__5_0] []
[2019-01-18 15:20:43.7309] [Microsoft.AspNetCore.Diagnostics.Internal.DiagnosticsLoggerExtensions.UnhandledException] [Error]: [An unhandled exception has occurred while executing the request.] [] [AsyncMethodBuilderCore.Start => <Invoke>d__7.MoveNext => DiagnosticsLoggerExtensions.UnhandledException] []
[2019-01-18 15:25:42.7751] [WebCore.Startup+<>c__DisplayClass5_0.<Configure>b__9] [Info]: [网站启动] [] [CancellationTokenSource.ExecuteCallbackHandlers => ExecutionContext.RunInternal => <>c__DisplayClass5_0.<Configure>b__9] []
[2019-01-18 15:25:44.1615] [Microsoft.AspNetCore.Server.Kestrel.Https.Internal.HttpsConnectionAdapter+<InnerOnConnectionAsync>d__10.MoveNext] [Debug]: [Failed to authenticate HTTPS connection.] [] [AsyncStateMachineBox`1.MoveNext => ExecutionContext.RunInternal => <InnerOnConnectionAsync>d__10.MoveNext] []
[2019-01-18 15:25:44.1615] [Microsoft.AspNetCore.Server.Kestrel.Https.Internal.HttpsConnectionAdapter+<InnerOnConnectionAsync>d__10.MoveNext] [Debug]: [Failed to authenticate HTTPS connection.] [] [AsyncStateMachineBox`1.MoveNext => ExecutionContext.RunInternal => <InnerOnConnectionAsync>d__10.MoveNext] []
[2019-01-18 15:25:44.9194] [WebCore.Hosting.MyRabbitMqService.<StartAsync>b__5_0] [Info]: [RabbitMq开始监听] [] [Task.ExecuteWithThreadLocal => ExecutionContext.RunInternal => MyRabbitMqService.<StartAsync>b__5_0] []
[2019-01-18 15:25:53.2885] [Microsoft.AspNetCore.Diagnostics.Internal.DiagnosticsLoggerExtensions.UnhandledException] [Error]: [An unhandled exception has occurred while executing the request.] [] [AsyncMethodBuilderCore.Start => <Invoke>d__7.MoveNext => DiagnosticsLoggerExtensions.UnhandledException] []
[2019-01-18 15:35:13.0313] [WebCore.Startup+<>c__DisplayClass5_0.<Configure>b__9] [Info]: [网站启动] [] [CancellationTokenSource.ExecuteCallbackHandlers => ExecutionContext.RunInternal => <>c__DisplayClass5_0.<Configure>b__9] []
[2019-01-18 15:35:14.0217] [Microsoft.AspNetCore.Server.Kestrel.Https.Internal.HttpsConnectionAdapter+<InnerOnConnectionAsync>d__10.MoveNext] [Debug]: [Failed to authenticate HTTPS connection.] [] [AsyncStateMachineBox`1.MoveNext => ExecutionContext.RunInternal => <InnerOnConnectionAsync>d__10.MoveNext] []
[2019-01-18 15:35:14.9641] [WebCore.Hosting.MyRabbitMqService.<StartAsync>b__5_0] [Info]: [RabbitMq开始监听] [] [Task.ExecuteWithThreadLocal => ExecutionContext.RunInternal => MyRabbitMqService.<StartAsync>b__5_0] []
[2019-01-18 15:35:27.6574] [Microsoft.AspNetCore.Diagnostics.Internal.DiagnosticsLoggerExtensions.UnhandledException] [Error]: [An unhandled exception has occurred while executing the request.] [] [AsyncMethodBuilderCore.Start => <Invoke>d__7.MoveNext => DiagnosticsLoggerExtensions.UnhandledException] []
[2019-01-18 15:38:32.2550] [WebCore.Startup+<>c__DisplayClass5_0.<Configure>b__9] [Info]: [网站启动] [] [CancellationTokenSource.ExecuteCallbackHandlers => ExecutionContext.RunInternal => <>c__DisplayClass5_0.<Configure>b__9] []
[2019-01-18 15:38:33.4889] [Microsoft.AspNetCore.Server.Kestrel.Https.Internal.HttpsConnectionAdapter+<InnerOnConnectionAsync>d__10.MoveNext] [Debug]: [Failed to authenticate HTTPS connection.] [] [HttpsConnectionAdapter.InnerOnConnectionAsync => AsyncMethodBuilderCore.Start => <InnerOnConnectionAsync>d__10.MoveNext] []
[2019-01-18 15:38:34.1155] [WebCore.Hosting.MyRabbitMqService.<StartAsync>b__5_0] [Info]: [RabbitMq开始监听] [] [Task.ExecuteWithThreadLocal => ExecutionContext.RunInternal => MyRabbitMqService.<StartAsync>b__5_0] []
[2019-01-18 15:38:45.1551] [Microsoft.AspNetCore.Diagnostics.Internal.DiagnosticsLoggerExtensions.UnhandledException] [Error]: [An unhandled exception has occurred while executing the request.] [] [AsyncMethodBuilderCore.Start => <Invoke>d__7.MoveNext => DiagnosticsLoggerExtensions.UnhandledException] []
[2019-01-18 15:40:31.4849] [WebCore.Startup+<>c__DisplayClass5_0.<Configure>b__9] [Info]: [网站启动] [] [CancellationTokenSource.ExecuteCallbackHandlers => ExecutionContext.RunInternal => <>c__DisplayClass5_0.<Configure>b__9] []
[2019-01-18 15:40:32.9746] [Microsoft.AspNetCore.Server.Kestrel.Https.Internal.HttpsConnectionAdapter+<InnerOnConnectionAsync>d__10.MoveNext] [Debug]: [Failed to authenticate HTTPS connection.] [] [HttpsConnectionAdapter.InnerOnConnectionAsync => AsyncMethodBuilderCore.Start => <InnerOnConnectionAsync>d__10.MoveNext] []
[2019-01-18 15:40:33.0024] [Microsoft.AspNetCore.Server.Kestrel.Https.Internal.HttpsConnectionAdapter+<InnerOnConnectionAsync>d__10.MoveNext] [Debug]: [Failed to authenticate HTTPS connection.] [] [HttpsConnectionAdapter.InnerOnConnectionAsync => AsyncMethodBuilderCore.Start => <InnerOnConnectionAsync>d__10.MoveNext] []
[2019-01-18 15:40:34.1737] [WebCore.Hosting.MyRabbitMqService.<StartAsync>b__5_0] [Info]: [RabbitMq开始监听] [] [Task.ExecuteWithThreadLocal => ExecutionContext.RunInternal => MyRabbitMqService.<StartAsync>b__5_0] []
[2019-01-18 15:40:42.9082] [Microsoft.AspNetCore.Diagnostics.Internal.DiagnosticsLoggerExtensions.UnhandledException] [Error]: [An unhandled exception has occurred while executing the request.] [] [AsyncMethodBuilderCore.Start => <Invoke>d__7.MoveNext => DiagnosticsLoggerExtensions.UnhandledException] []
[2019-01-18 15:41:39.5528] [WebCore.Startup+<>c__DisplayClass5_0.<Configure>b__9] [Info]: [网站启动] [] [CancellationTokenSource.ExecuteCallbackHandlers => ExecutionContext.RunInternal => <>c__DisplayClass5_0.<Configure>b__9] []
[2019-01-18 15:41:40.7612] [Microsoft.AspNetCore.Server.Kestrel.Https.Internal.HttpsConnectionAdapter+<InnerOnConnectionAsync>d__10.MoveNext] [Debug]: [Failed to authenticate HTTPS connection.] [] [HttpsConnectionAdapter.InnerOnConnectionAsync => AsyncMethodBuilderCore.Start => <InnerOnConnectionAsync>d__10.MoveNext] []
[2019-01-18 15:41:40.7612] [Microsoft.AspNetCore.Server.Kestrel.Https.Internal.HttpsConnectionAdapter+<InnerOnConnectionAsync>d__10.MoveNext] [Debug]: [Failed to authenticate HTTPS connection.] [] [HttpsConnectionAdapter.InnerOnConnectionAsync => AsyncMethodBuilderCore.Start => <InnerOnConnectionAsync>d__10.MoveNext] []
[2019-01-18 15:41:42.0328] [WebCore.Hosting.MyRabbitMqService.<StartAsync>b__5_0] [Info]: [RabbitMq开始监听] [] [Task.ExecuteWithThreadLocal => ExecutionContext.RunInternal => MyRabbitMqService.<StartAsync>b__5_0] []
[2019-01-18 15:41:59.9020] [Microsoft.AspNetCore.Diagnostics.Internal.DiagnosticsLoggerExtensions.UnhandledException] [Error]: [An unhandled exception has occurred while executing the request.] [] [AsyncMethodBuilderCore.Start => <Invoke>d__7.MoveNext => DiagnosticsLoggerExtensions.UnhandledException] []
